src/auto/file.rs

```diff
@@ -205,12 +205,12 @@ pub trait FileExt: 'static {
 
     fn make_symbolic_link<P: AsRef<std::path::Path>, Q: IsA<Cancellable>>(&self, symlink_value: P, cancellable: Option<&Q>) -> Result<(), Error>;
 
-    //fn measure_disk_usage<P: IsA<Cancellable>>(&self, flags: /*Ignored*/FileMeasureFlags, cancellable: Option<&P>, progress_callback: Option<Box<dyn Fn(bool, u64, u64, u64) + 'static>>) -> Result<(u64, u64, u64), Error>;
+    fn measure_disk_usage<P: IsA<Cancellable>>(&self, flags: FileMeasureFlags, cancellable: Option<&P>, progress_callback: Option<Box<dyn Fn(bool, u64, u64, u64) + 'static>>) -> Result<(u64, u64, u64), Error>;
 
-    //fn measure_disk_usage_async<P: IsA<Cancellable>, Q: FnOnce(Result<(u64, u64, u64), Error>) + Send + 'static>(&self, flags: /*Ignored*/FileMeasureFlags, io_priority: glib::Priority, cancellable: Option<&P>, progress_callback: Option<Box<dyn Fn(bool, u64, u64, u64) + 'static>>, callback: Q);
+    //fn measure_disk_usage_async<P: IsA<Cancellable>, Q: FnOnce(Result<(u64, u64, u64), Error>) + Send + 'static>(&self, flags: FileMeasureFlags, io_priority: glib::Priority, cancellable: Option<&P>, progress_callback: Option<Box<dyn Fn(bool, u64, u64, u64) + 'static>>, callback: Q);
```

Comment:
Why is this not generated still? Seems like nothing is actually missing now

Reply:
I wonder...

Reply:
`measure_disk_usage_async: missing success parameters for async future`

Reply:
What does that mean? :)

Reply:
Aren't you the one who wrote this part of gir? :p Or maybe @antoyo?

Reply:
Not me :)

Reply:
@GuillaumeGomez: Looks like [you](https://github.com/gtk-rs/gir/blame/56f6de49e185fb3ff8502ed0d4ddf447d8d5fc05/src/analysis/functions.rs#L725) did it.

Reply:
Ok, maybe it's on me.... Again... XD

Reply:
Let's create a gir issue for this then? :)

Reply:
https://github.com/gtk-rs/gir/issues/783